crates/hir-expand/src/name.rs

```diff
@@ -70,6 +70,8 @@ impl PartialEq<Name> for &Symbol {
 }
 
 impl Name {
+    pub const MISSING: &Name = &Name { symbol: sym::consts::missing, ctx: () };
```

Comment:
We have a `Name::missing` function that we can replace with that const

Reply:
I did the opposite, removed the const and made the function `const fn`.